backend/api/Services/RobotService.cs

```diff
@@ -163,6 +165,39 @@ public async Task<IList<Robot>> ReadLocalizedRobotsForInstallation(string instal
                 .ToListAsync();
         }
 
+        public async Task OnIsarUnavailable(string robotId)
```

Comment:
Consider moving to an event handler. Could be RobotEventHandler or something descriptive. If it is to be in the service it should be renamed so id does not sound like something that handles an event

Reply:
Given that it is not called in the sequence of an event I kept it in `RobotService` and renamed it to `SetRobotOffline`